assets/js/modules/idea-hub/datastore/idea-state.js

```diff
@@ -20,6 +20,7 @@
  * External dependencies
  */
 import invariant from 'invariant';
+import { omit } from 'lodash';
```

Comment:
Let's import the omit function from `lodash/omit`:
```suggestion
import omit from 'lodash/omit';
```